### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by Phenomenalhub**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <resources>
     <string name="app_name">StudentArena</string>
     <string name="Post_Item">Post Item</string>
+    <string name="You">You</string>
+    <string name="Log_out">Log out</string>
+    <string name="Full_name">Full name</string>
```
Comment (sushant10) [inline app/src/main/res/values/strings.xml:6]: I think we usually try to follow lower case for the string names

File: app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java [hunk 2]
```diff
@@ -52,19 +53,16 @@ public int getItemCount() {
...
             super(itemView);
             ivImage = itemView.findViewById(R.id.ivImage);
-            //tvPrice = itemView.findViewById(R.id.tvPrice);
             tvDetail = itemView.findViewById(R.id.tvTitle);
         }
 
         public void bind(Post post) {
             // Bind the post data to the view elements
             tvDetail.setText("$" + post.getPrice()+ " •  " + post.getTitle());
-            //tvPrice.setText("$" + post.getPrice());
```
Comment (sushant10) [inline app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java]: Thanks for deleting the commented code!

File: app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    class ViewHolder extends RecyclerView.ViewHolder {
+        private ImageView ivImage;
+        private TextView tvPrice;
+        private TextView tvDetail;
+        public ViewHolder(@NonNull View itemView) {
+            super(itemView);
+            ivImage = itemView.findViewById(R.id.ivImage);
+            //tvPrice = itemView.findViewById(R.id.tvPrice);
```
Comment (sushant10) [inline app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java]: remove?

File: app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            tvDetail = itemView.findViewById(R.id.tvTitle);
+        }
+
+        public void bind(Post post) {
+            // Bind the post data to the view elements
+            tvDetail.setText("$" + post.getPrice()+ " •  " + post.getTitle());
+            //tvPrice.setText("$" + post.getPrice());
```
Comment (sushant10) [inline app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java]: same here

File: app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                ivImage.setVisibility(View.VISIBLE);
+                Glide.with(context).load(image.getUrl()).transform(new CenterCrop(),new RoundedCorners(25)).into(ivImage);
+            } else{
+                ivImage.setVisibility(View.GONE);
+            }
+            ivImage.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    Intent i = new Intent(context, DetailActivity.class);
+
```
Comment (sushant10) [inline app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java:77]: format -> remove space

File: app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                    i.putExtra("Posts", Parcels.wrap(post));
```
Comment (sushant10) [inline app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java:78]: maybe we can store intent extras constant to another file. The advantage being, if you ever want to rename an extra you dont have to go looking for every occurence of it but instead change the single source of truth file

File: app/src/main/java/com/example/studentarena/fragments/ProfileFragment.java [hunk 7]
```diff
@@ -97,9 +98,11 @@ public void onClick(View v) {
             Glide.with(view).load(userProfileImg.getUrl()).centerCrop().circleCrop().override(Target.SIZE_ORIGINAL, Target.SIZE_ORIGINAL).into(ivProfileImage);
         };
 
-        rvPosts =view.findViewById(R.id.rvPosts);
-        StaggeredGridLayoutManager gridLayoutManager = new StaggeredGridLayoutManager(2, StaggeredGridLayoutManager.VERTICAL);
-        rvPosts.setLayoutManager(gridLayoutManager);
+        rvProfile =view.findViewById(R.id.rvProfile);
+        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(getContext());
+        //StaggeredGridLayoutManager gridLayoutManager = new StaggeredGridLayoutManager(2, StaggeredGridLayoutManager.VERTICAL);
```
Comment (sushant10) [inline app/src/main/java/com/example/studentarena/fragments/ProfileFragment.java]: remove

**Review by sushant10 — APPROVED** [review]
Small changes here and there. Approving to unblock

File: app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java (hunk 3 above)
Comment (Phenomenalhub) [reply app/src/main/java/com/example/studentarena/adapter/ProfileAdapter.java]: I deleted all comments in my next commit.Thanks

**Pull request merged**